FordLabs/PeopleMover
# Read-only View

## Issue
Resolves #377 

## What was done
- [x] Disallow access to any post, put or delete endpoint in read only view
- [x] Disallow access to get endpoints in read only view that have a requested date other than today
- [x] Disable UI for editing a space in read only view

## How to test
1. Go to a space not in your space dashboard
2. Make sure you cannot edit that space in anyway
3. Make sure you cannot view data for dates other than today

### FOR DEVS
Feature URL: See feature branch

### FOR APPROVAL
Approval URL: See Approval branch
